docs/deprecations.md

```diff
@@ -386,3 +386,11 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 Deprecation code: `BIN_SERVERLESS`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
```

Comment:
Let's put the deprecation details at the top of the file, right below the instructions on how to disable specific deprecations